docs/build/get-started.mdx

```diff
@@ -270,6 +81,17 @@ Choose the *minimal* scaffolding option, and press return. When prompted, enter
 Successfully initialized Fluence project template at ~/localdev/hello-world
 ```
 
+Also, we can choose across different networks to deploy our services. For now, we will use the default network, which is `kras`:
+
+```bash
+? Select Fluence Environment to use by default with this project (Use arrow keys)
+❯ kras (default)
+  testnet
+  stage
+  local
```

Comment:
Why is local red? Maybe remove `bash`? confuses focus

Reply:
fixed